cli/src/semdep/parse_lockfile.py

```diff
@@ -140,7 +140,11 @@ def parse_package_lock(
     else:
         logger.debug("Found package-lock with no 'dependencies'")
         return
-
+    if manifest_text:
```

Comment:
Hmm looks like this is a duplicate?

Reply:
No, this one is in `parse_package_lock`, the other is in `parse_yarn`.

Reply:
Wouldnt the lines on 149 below still cause this to crash